code/game/objects/items/devices/radio/radio.dm

```diff
@@ -656,21 +657,32 @@ var/GLOBAL_RADIO_TYPE = 1 // radio type to use
 		to_chat(user, "<span class='notice'>\the [src] can[b_stat ? "" : " not"] be attached or modified!</span>")
 
 /obj/item/device/radio/attackby(obj/item/weapon/W, mob/user)
-	if (istype(W, /obj/item/weapon/screwdriver))
+	if(istype(W, /obj/item/device/radio_grid) && !grid)

```

Comment:
Эх, наверно придется еще третий пост сотворить, т.к поди иначе нотиса не будет, а я только сейчас осознал что ты скинул ссылку на то, чем никто в общем-то не пользуется, из-за того что там как-то очень хреново это реализовано и половину информации куда-то сжевали.

@Gyaviste пользуйся вот этим - http://www.byond.com/docs/ref/info.html#/proc/if - вот тебе на if else if кстати. А вот ссылка на это в целом на нормальные доки - http://www.byond.com/docs/ref/

И собственно то что важно там и написано - `If the expression E is true (non-zero) then execute Statement1. Otherwise, test E2, etc. Finally, if none of the expressions are true, execute Statement3. The else nodes are all optional. `